projects/standard-rulesets/src/breaking-changes/preventArrayItemLengthChanges.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  RequestRule,
+  ResponseBodyRule,
+  RuleError,
+} from '@useoptic/rulesets-base';
+import {
+  arrayMaxItemsShouldNotIncrease,
+  arrayMinItemsShouldNotDecrease,
+  numericMaximumRangeShouldNotExpand,
+  numericMinimumRangeShouldNotExpand,
+} from './helpers/rangeChange';
```

Comment:
nit: Can we move these functions into this file? The implementations aren't generic enough to need to be used elsewhere since they contain both the logic of whether a schema has a range increase, and the rule error

Reply:
to confirm, just the array ones, right? The numeric ones are used in multiple places

Reply:
It looks like the numeric ones aren't used in this file.